### PR TITLE
Add Tweak for making the Follow button more subtle

### PR DESCRIPTION
**Pull request by cyle**
This adds a new Tweak to make the Follow button more subtle.

The existing design:

Screen Shot 2020-07-03 at 3 07 13 PM

With this tweak, I just reset it to the `--gray-40` Tumblr official palette variable, which looks great (to me) across palettes:

Screen Shot 2020-07-03 at 3 12 30 PM

Screen Shot 2020-07-03 at 3 12 38 PM

Screen Shot 2020-07-03 at 3 12 44 PM

hello again @AprilSylph @nightpool @invalidCards  😄

### CODE REVIEW

File: Extensions/tweaks.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Tweaks **//
-//* VERSION 6.0.1 **/
+//* VERSION 6.0.3 **/
```
Comment (cyle) [inline Extensions/tweaks.js:2]: yeah i gotta update this :eyes:

File: Extensions/tweaks.js [hunk 2]
```diff
@@ -786,11 +792,11 @@ XKit.extensions.tweaks = new Object({
 				${notificationBadgeStyle}
 			}`, 'xkit_tweaks_notification_badge_style');
 		}
-		
+
```
Comment (cyle) [inline Extensions/tweaks.js:795]: whitespace changes were picked up by Atom applying the repo's `.editorconfig`, fyi

File: Extensions/tweaks.js [hunk 3]
```diff
@@ -801,14 +807,21 @@ XKit.extensions.tweaks = new Object({
...
 		if (XKit.extensions.tweaks.preferences.grayscale_new_post_button.value) {
 			let postIconButtonSel = XKit.css_map.keyToClasses('postIconButton').map(cssClass => `.${cssClass} span`).join(',');
 			XKit.extensions.tweaks.add_css(`${postIconButtonSel} {
 				filter: grayscale(100%);
 			}`, 'xkit_tweaks_grayscale_new_post_button');
 		}
-		
+
+		if (XKit.extensions.tweaks.preferences.subtle_follow_button.value) {
+			let followButtonSelector = XKit.css_map.keyToClasses('followButton').map(cssClass => `.${cssClass}`).join(',');
```
Comment (invalidCards) [inline Extensions/tweaks.js:819]: You can just use `keyToCss` here, since that does all the mapping work for you

Comment (cyle) [reply Extensions/tweaks.js:819]: ah i had tried that and it didn't work... lemme try again i guess

Comment (cyle) [reply Extensions/tweaks.js]: ah i was using it incorrectly before. updated!

**Comment by cyle** [comment]
Updated to make this inside-posts-only so as to mess with the weird side effects of follow buttons elsewhere 👌

**Pull request merged**